hypercert.json

```diff
@@ -0,0 +1,113 @@
+{
+  "lexicon": 1,
+  "id": "app.certified.hypercert.record",
+  "defs": {
+    "main": {
+      "type": "record",
+      "description": "A hypercert record tracking impact work.",
+      "key": "any",
+      "record": {
+        "type": "object",
+        "required": ["hypercertID", "title", "description", "image", "createdAt", "workScope", "workTimeframeFrom", "workTimeFrameTo"],
+        "properties": {
+          "title": {
+            "type": "string",
+            "description": "Title of the hypercert",
+            "maxLength": 256
+          },
+          "hypercertID": {
```

Comment:
Curious why we need an extra id here? Isn't the record key sufficient? IIUC, the protocol itself will generate a CID so we don't need a property for that. Then the hypercert can be uniquely identified and discovered via the strongRef.